### PR TITLE
Update the requirements for the Linguist project

### PR DESCRIPTION
**Pull request by vmg**
The proposed Linguist projects have been unified and specified with more detail. We're now making mention of memory/performance requirements, pointing out that Ruby is not a hard requirement for the project, and hinting at the complexity of the task -- not in order to scare away prospective students, but in hopes of attracting an amazing student who would be up to the task.

cc @bkeepers @mikemcquaid @arfon

### CODE REVIEW

**Comment by arfon** [comment]
:+1:

**Comment by MikeMcQuaid** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -136,25 +136,27 @@ Mentor: [@mislav](https://github.com/mislav)
...
+We're looking for ways to *increase the accuracy of the language classifier* and at the same time *reduce its memory usage and runtime*. For your proposal, discuss possible approaches towards improving accuracy (either by using novel machine learning techniques or by improving the existing approaches and/or dataset) and towards reducing the memory footprint of the classification. Ruby is not a hard requirement for the project -- we're looking for efficient solutions in Systems programming languages (C, C++, Go...) or for improvements on the design of the Ruby solution.
 
-Mentors: [@arfon](https://github.com/arfon), [@bkeepers](https://github.com/bkeepers)
+Expected Outcome: A classifier that is both more accurate, faster and uses less memory than the original Ruby implementation. **This is a hard (and rewarding) project**.
 
-Skills: Ruby, language learning
+Mentors: [@vmg](https://github.com/vmg), [@arfon](https://github.com/arfon), [@bkeepers](https://github.com/bkeepers)
+
+Skills: Basic machine learning knowledge, solid knowledge of a Systems programming language or Ruby
```
Comment (MikeMcQuaid) [inline README.md:159]: tiny nit but Systems being capitalised seems weird? Maybe just me.

**Comment by bkeepers** [comment]
:shipit:

**Comment by vmg** [comment]
Thanks guys! Really appreciated. :)))

File: README.md (hunk 1 above)
Comment (vmg) [inline README.md:159]: Oops, missed this comment. I guess it's like GitHub style? I don't know, it feels like we've always capitalized Systems internally. :)

Comment (MikeMcQuaid) [inline README.md:159]: Cool, I'm easy either way :+1:

**Pull request merged**